plugins/anthropic/claude_code.go

```diff
@@ -0,0 +1,22 @@
+package anthropic
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func ClaudeCodeCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Claude Code",
+		Runs:      []string{"claude"},
+		DocsURL:   sdk.URL("https://docs.anthropic.com/en/docs/agents-and-tools/claude-code/overview"),
+		NeedsAuth: needsauth.NotForHelpOrVersion(),
+		Uses: []schema.CredentialUsage{
+			{
+				Name: credname.APIKey,
+			},
+		},
+	}
```

Comment:
Let run `gofmt -w .` in this directory, to make sure the code passes the linter.

Happy to reapprove and merge afterwards!

Reply:
```shell
~/C/s/plugins/anthropic on add-anthropic
❯ gofmt -w .

~/C/s/plugins/anthropic on add-anthropic
❯ git s
On branch add-anthropic
Your branch is up to date with 'origin/add-anthropic'.

nothing to commit, working tree clean
```